azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -348,4 +352,19 @@ private JobRunner createJobRunner(final int execId, final String name, final int
     return runner;
   }
 
+  private void waitForStatus(ExecutableNode node, Status status) {
```

Comment:
This method looks the same as https://github.com/HappyRay/azkaban/blob/96ad15a98af88adb1c17b7486802ed016bd109f1/azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTestBase.java#L120
It would be great to consolidate them.